PF/PFAssignment3/StringOperations/StringOperations.java

```diff
@@ -0,0 +1,134 @@
+package GET2018.PF.PFAssignment3.StringOperations;
+
+/**
+ * This class contains different operations applied on String
+ * 
+ * @author Mohit Sharma
+ *
+ */
+public class StringOperations {
+	private String operatedString;
+
+	public StringOperations(String operatedString) throws StringException {
+		if (operatedString == null) {
+			throw new NullPointerException("String is null");
+		} else if (operatedString.length() == 0) {
+			throw new StringException("String is Empty");
+		}
+		this.operatedString = operatedString;
+	}
+
+	/**
+	 * Checks whether Entered String is equal to given String
+	 * 
+	 * @param checkEqualityString
+	 * @return 1 if both strings are equal otherwise 0
+	 * @throws StringException
+	 */
+	public int isEquals(String checkEqualityString) throws StringException {
```

Comment:
I believe return type of this method should be boolean.
Also there should not be more than 1 or 2 return statements. Maybe you can store return value in a variable.

Reply:
Ma'am in the Question it is stated that return 1 if equals else 0.